live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/adapters/config/IntegrationConfig.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * #%L
+ * Alfresco HX Insight Connector
+ * %%
+ * Copyright (C) 2024 Alfresco Software Limited
+ * %%
+ * This file is part of the Alfresco software.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail.  Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * Alfresco is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Alfresco is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ * #L%
+ */
+package org.alfresco.hxi_connector.live_ingester.adapters.config;
+
+import jakarta.validation.constraints.NotNull;
+
+import lombok.Data;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.boot.context.properties.NestedConfigurationProperty;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.validation.annotation.Validated;
+
+import org.alfresco.hxi_connector.live_ingester.adapters.config.properties.Ingester;
+import org.alfresco.hxi_connector.live_ingester.adapters.config.properties.Repository;
+import org.alfresco.hxi_connector.live_ingester.adapters.config.properties.Storage;
+import org.alfresco.hxi_connector.live_ingester.adapters.config.properties.Transform;
+
+@Configuration
+@EnableConfigurationProperties
+public class IntegrationConfig
```

Comment:
Do we need this class? It seems like we're using the `Properties` inner class everywhere, which suggests that would be more useful as a class of it's own using the path `alfresco.integration.properties`.

Reply:
I had issues with this approach earlier due two beans, which Spring created:
`alfresco.integration-org.alfresco.hxi_connector.live_ingester.adapters.config.IntegrationProperties`
`integrationProperties`
but right now it seems to work fine...
Possibly I had some misconfiguration related with `@EnableConfigurationProperties` or `@ConfigurationPropertiesScan` 🤔 

As it works fine I've fixed that according to your suggestion.